### DISCUSSION TITLE
Page break for MS Word and  PDF output

### DISCUSSION THREAD

**Discussion by Pierre9344** [quarto-dev/quarto-cli#2903] [Q&A] [answered]
Hello,

Is there a way to indicate a page or section break in quarto?
Also, is there a way to control the size of the equation rendered?

**Comment by mcanouil** [comment, marked as answer]
Hi,

`\pagebreak` and/or `\newpage` should work for both PDF and Word.

Please take a look at #2170 and #2392, these should answer your second question.

**Reply by Pierre9344** [reply]
Thank you for the answer. The \pagebreak and \newpage work. However, the equation size isn't modified with the css it probably isn't used as I render to MS Word...

**Reply by mcanouil** [reply]
For Word, you'll have to define a Word template where you tweak the style associated with math.

**Reply by mcanouil** [reply]
For LaTeX, the usual commands such as `\large`, `\small`, etc. should work, but I don't know if the code will work on all formats.

**Reply by Pierre9344** [reply]
Thank you for the information
